tests/helpers.py

```diff
@@ -306,13 +328,43 @@ def create_ceph_file_system(pool_name=None):
     return cfs_data
 
 
+def default_storage_class(
+    interface_type,
+):
+    """
+    Return default storage class based on interface_type
+
+    Args:
+        interface_type (str): The type of the interface
+            (e.g. CephBlockPool, CephFileSystem)
+
+    Returns:
+        OCS: Existing StorageClass Instance
+    """
+
+    if interface_type == constants.CEPHBLOCKPOOL:
+        base_sc = OCP(
+            kind='storageclass',
+            resource_name=constants.DEFAULT_STORAGECLASS_RBD
+        )
+    elif interface_type == constants.CEPHFILESYSTEM:
+        base_sc = OCP(
+            kind='storageclass',
+            resource_name=constants.DEFAULT_STORAGECLASS_CEPHFS
+        )
```

Comment:
elif openshift-storage.noobaa.io ?

Reply:
I will take this up as follow on PR, will force me to retest with any new code changes(2 PRs)